dev/WindowsAppRuntime.ProxyStub/WindowsAppRuntime.ProxyStub.vcxproj

```diff
@@ -161,6 +159,34 @@
     <LinkIncremental>false</LinkIncremental>
     <CustomBuildAfterTargets>PostBuildEvent</CustomBuildAfterTargets>
   </PropertyGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
```

Comment:
Where'd this come from?

This is doing the hybrid CRT game, but we shouldn't be mucking with that in individual projects anymore. \Directory.build.props imports \HybridCRT.props with all the right settings so everything in the repository gets hybrid CRT support.

Remove this block and restore line 45 ?